Yoast/YoastSEO.js
# Create readability score aggregator

## Summary

This PR can be summarized in the following changelog entry:

* _[non-user-facing]_ Adds a `ReadabilityScoreAggregator` that should serve as a replacement for the score aggregation part of the current `contentAssessor` in the (near) future.

## Relevant technical choices:

* Uses the same logic as in `contentAssessor`.
* May get some merge conflicts when #2173 is merged first, since it restructures the folder structure somewhat. These changes may not coincide with the structure in this branch.

## Test instructions

This PR can be tested by following these steps:

* Look at the code.
    * Specifically, check if the logic used is the same as the score aggregation functionality in `contentAssessor` (The `aggregate` method in the `ReadabilityScoreAggregator` method should implement the same logic as the `calculateOverallScore` method).
 * Run the tests.
    * Check that the coverage is 100% for the `ReadabilityScoreAggregator`.
    * Check that all tests pass.

Fixes #2125
